internal/artifact/run.go

```diff
@@ -66,6 +66,11 @@ func run(c config.Config, initializeScanner InitializeScanner) error {
 	}
 
 	ctx := context.Background()
+	if c.Timeout > 0 {
+		var cancel context.CancelFunc
+		ctx, cancel = context.WithTimeout(ctx, c.Timeout)
+		defer cancel()
+	}
```

Comment:
`Timeout` has the default value.
https://github.com/aquasecurity/trivy/blob/020c4a3b140b3ee329eba8926de79e2889ec9312/internal/app.go#L155

How about the following way?

```suggestion
	ctx := context.WithTimeout(context.Background(), c.Timeout)
	defer cancel()
```